ipfs/ipfs.go

```diff
@@ -0,0 +1,316 @@
+package ipfs
+
+import (
+	"context"
+	"errors"
+	"flag"
+	"fmt"
+	"io/fs"
+	"io/ioutil"
+	"log"
+	"os"
+	"path/filepath"
+	"sync"
+
+	config "github.com/ipfs/go-ipfs-config"
+	files "github.com/ipfs/go-ipfs-files"
+	iface "github.com/ipfs/interface-go-ipfs-core"
+	ifacepath "github.com/ipfs/interface-go-ipfs-core/path"
+	ma "github.com/multiformats/go-multiaddr"
+
+	"github.com/ipfs/go-ipfs/core"
+	"github.com/ipfs/go-ipfs/core/coreapi"
+	"github.com/ipfs/go-ipfs/core/node/libp2p"
+	"github.com/ipfs/go-ipfs/plugin/loader" // This package is needed so that all the preloaded plugins are loaded automatically
+	"github.com/ipfs/go-ipfs/repo/fsrepo"
+	"github.com/libp2p/go-libp2p-core/peer"
+	status "github.com/sonr-io/core/errors"
+)
+
+var FlagExp = flag.Bool("experimental", false, "enable experimental features")
```

Comment:
This name seems a little wordier than I would like, but I do like it over FlagExp, which is a name I just stole off of their example. Either way I don't see us wanting to use experimental features for the time being. In my mind I imagined this would be for dev use only, if I'm wrong in that assumption than I think making this an env variable would be better to make it configurable. 

This flag just says if you want to use any experminetal features and if enabled just turns all of them on. We could make this process more sophisicated if we think it would be useful.

Reply:
Okay lgtm